External/Plugins/ProjectManager/Controls/FDMenus.cs

```diff
@@ -182,16 +182,16 @@ public void ToggleDebugRelease()
     /// </summary>
     public class ProjectMenu : ToolStripMenuItem
     {
-        public ToolStripMenuItemEx NewProject;
-        public ToolStripMenuItemEx OpenProject;
-        public ToolStripMenuItemEx ImportProject;
-        public ToolStripMenuItemEx CloseProject;
-        public ToolStripMenuItemEx OpenResource;
-        public ToolStripMenuItemEx TestMovie;
-        public ToolStripMenuItemEx RunProject;
-        public ToolStripMenuItemEx BuildProject;
-        public ToolStripMenuItemEx CleanProject;
-        public ToolStripMenuItemEx Properties;
+        public ToolStripMenuItem NewProject;
+        public ToolStripMenuItem OpenProject;
+        public ToolStripMenuItem ImportProject;
+        public ToolStripMenuItem CloseProject;
+        public ToolStripMenuItem OpenResource;
+        public ToolStripMenuItem TestMovie;
+        public ToolStripMenuItem RunProject;
+        public ToolStripMenuItem BuildProject;
+        public ToolStripMenuItem CleanProject;
+        public ToolStripMenuItem Properties;
```

Comment:
This type of rollback should be reverted for the next major update, when the compatibility is not needed.
Since the `ToolStripMenuItexEx.ShortcutKeys` is not an overridden member, every time you need to use `ShortcutKeys` on these, you need to cast them to `ToolStripMenuItemEx`.